common/changes/@cadl-lang/compiler/feature-importcompletion_2022-06-07-14-45.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/compiler",
+      "comment": "?",
```

Comment:
this should an entry describing the change. `Add completion  support for import of libraries`